### PR TITLE
Add block generator via nil_cli

### PR DESCRIPTION
**Pull request by makxenov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -194,6 +194,22 @@ Test block could be generated from config file in JSON format
...
 
+### Nil CLI block generation
+
+Wrapper script for `nil_cli` is available to deploy and call Solidity contracts.
+It requires `nil` and `nil_cli` binaries in `PATH` (already available inside Nix dev environment).
+
+Usage:
+
+```bash
+# Generate bytecode and abi files for contract
```
Comment (akokoshn) [inline README.md:205]:
```
# Automatically deploy and call contracts with using config
./bin/block_generator/block_generator.py --mode generate-block --config-name test_config.yaml
```
Add link to the config exemple
```
# For manually deploy and call contracts use `nil_cli `directly (https://github.com/NilFoundation/nil/README.md)
```

Comment (makxenov) [reply README.md]: Added

File: bin/block_generator/block_generator.py [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+import logging
+from pathlib import Path
+import random
+import re
+import subprocess
+from time import sleep
+from typing import Dict
+
+# Interface for launching nil cluster RPC server
+class ClusterProcess:
```
Comment (akokoshn) [inline bin/block_generator/block_generator.py:17]: `RPCService`

**Review by akokoshn — APPROVED** [review]

File: bin/block_generator/block_generator.py (hunk 2 above)
Comment (makxenov) [reply bin/block_generator/block_generator.py:17]: I think `RPC server` is also OK. Example: https://github.com/NilFoundation/nil/blob/main/rpc/transport/client.go#L20

**Pull request merged**